src/EditorFeatures/VisualBasicTest/Utils.vb

```diff
@@ -327,7 +327,7 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests
                         "End Structure"
 
                 Case "File"
-                    Return String.Join(vbCrLf, lines)
+                    Return testSource.NormalizedValue
```

Comment:
⚠️ This causes some test inputs to change (addition of blank lines at the beginning and/or end of the input), but I didn't see a clean way to separate the two.